core/micro-services/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/common/Utils.scala

```diff
@@ -0,0 +1,28 @@
+package edu.uci.ics.amber.operator.common
+
+import com.fasterxml.jackson.annotation.JsonInclude.Include
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.noctordeser.NoCtorDeserModule
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import com.typesafe.scalalogging.LazyLogging
+
+import java.nio.file.{Files, Path, Paths}
+import java.text.SimpleDateFormat
+import scala.annotation.tailrec
+
+object Utils extends LazyLogging {
```

Comment:
for all util classes, we need to give it a proper name, e.g, `XXXUtils`so that the naming is meaningful.